code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -296,7 +296,7 @@
 		if(!ishuman(target))
 			to_chat(user, "<span class='warning'>You can only enthrall humans.</span>")
 			break
-		if(do_mob(user, target, 50))
+		if(do_mob(user, target, 150))
```

Comment:
There's a `SECONDS` define you can use for better readabillity, which just converts deciseconds into seconds:
`#define SECONDS *10`
```suggestion
		if(do_mob(user, target, 15 SECONDS))
```